modules/custom/dkan_datastore/tests/src/Unit/Controller/DatastoreTest.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Drupal\Tests\dkan_datastore\Unit\Controller;
 
-use Drupal\dkan_datastore\Controller\Datastore;
+use Drupal\dkan_datastore\Controller\Api;
 use Dkan\PhpUnit\DkanTestBase;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Drupal\Core\DependencyInjection\ContainerBuilder;
 use Drupal\dkan_datastore\SqlParser;
 
 /**
- * @coversDefaultClass \Drupal\dkan_datastore\Controller\Datastore
+ * @coversDefaultClass \Drupal\dkan_datastore\Controller\Api
```

Comment:
If you're testing a different controller, you should rename the File itself from `DatastoreTest` -> `ApiTest`